ldap/servers/slapd/auditlog.c

```diff
@@ -177,6 +177,40 @@ write_auditfail_log_entry(Slapi_PBlock *pb)
     slapi_ch_free_string(&audit_config);
 }
 
+/*
+ * Write the attribute values to the audit log as "comments"
+ *
+ *   Slapi_Attr *entry - the attribute begin logged.
+ *   char *attrname - the attribute name.
+ *   lenstr *l - the audit log buffer
+ *
+ *   Resulting output in the log:
+ *
+ *       #ATTR: VALUE
+ *       #ATTR: VALUE
+ */
+static void
+log_entry_attr(Slapi_Attr *entry_attr, char *attrname, lenstr *l)
+{
+    Slapi_Value **vals = attr_get_present_values(entry_attr);
+    for(size_t i = 0; vals && vals[i]; i++) {
+        char log_val[256] = "";
+        struct berval *bv = slapi_value_get_berval(vals[i]);
```

Comment:
Should that be `const struct berval *bv = slapi_value_get_berval(vals[i]);`?

Reply:
Indeed, both you and the compiler are right ! -;)